main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -309,11 +310,18 @@ public void update(long bytesRead, long contentLength, int itemCount) {
                         CloseableHttpResponse response = httpclient.execute(httpGet);
 
                         try {
-                            response.getStatusLine();
+                            StatusLine status = response.getStatusLine();
```

Comment:
It would be tidier to put this with the rest of the status processing code below.